### PR TITLE
adding stack-hci-vm 1.0.0 version

### PR DESCRIPTION
**Pull request by hvedati**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by hvedati** [comment]
@jsntcy Could I get a review on this PR please?

File: src/index.json [hunk 1]
```diff
@@ -66838,6 +66838,59 @@
...
+                        "requires": [
+                            "azure-identity",
+                            "azure-identity",
+                            "azure-mgmt-resourcegraph",
+                            "azure-mgmt-resourcegraph"
+                        ]
+                        }
+                    ],
+                    "summary": "Microsoft Azure Command-Line Tools Stack-HCi-VM Extension",
+                    "version": "1.0.0.1"
```
Comment (jsntcy) [inline src/index.json:66891]: The version should contain three numbers: `major.minor.patch`.

Comment (hvedati) [reply src/index.json:66891]: Hi @jsntcy, This has been fixed! Could we please get a review on this PR?

**Review by ReaNAiveD — APPROVED** [review]

File: src/index.json (hunk 1 above)
Comment (jsntcy) [reply src/index.json:66891]: I think the version should be `1.0.0` as it's the first version starting from `1.x.x`.

Comment (hvedati) [reply src/index.json:66891]: The version has to be 1.0.1 since we had an internal release with a 1.0.0 whl file. We cannot override that file.

**Review by jsntcy — APPROVED** [review]

**Pull request merged**